chemper/graphs/fragment_graph.py

```diff
@@ -152,13 +152,15 @@ def as_smirks(self):
             return None
 
         if self.atom_by_smirks_index:
-            min_smirks = min(self.atom_by_smirks_index)
+            # sometimes we use negative numbers for internal indexing
```

Comment:
Why would negative indices be used?

Reply:
I'm using negative indices to track the location of non-indexed atoms. ChemicalEnvironments do this internally too where alpha or beta atoms are located by their negative smirks index. I probably need to add more information in the doc strings on what those values are allowed to be.

Reply:
I'm going to rename this as label, with details that positive labels are included in the SMIRKS patter, but that all other labels are just used as identification for storage.